client/src/pages/teds/index.tsx

```diff
@@ -1,9 +1,23 @@
-import PrctIpccReferencesByCountry from './charts/PrctIpccReferencesByCountry';
+import PrctIpccPublicationsByCountry from './charts/PrctIpccPublicationsByCountry';
+import PrctIpccPublicationsByCountryByWg1 from './charts/PrctIpccPublicationsByCountryByWg1';
+import PrctIpccPublicationsByCountryByWg2 from './charts/PrctIpccPublicationsByCountryByWg2';
+import PrctIpccPublicationsByCountryByWg2cross from './charts/PrctIpccPublicationsByCountryByWg2cross';
+import PrctIpccPublicationsByCountryByWg3 from './charts/PrctIpccPublicationsByCountryByWg3';
+
+import './gallery.css'
 
 export default function Welcome() {
   return (
-    <>
-      <PrctIpccReferencesByCountry />
-    </>
+    <div>
+      <div>
+        <PrctIpccPublicationsByCountry />
+      </div>
+      <div className="gallery">
```

Comment:
You should rather use the `dsfr-plus` lib to do that : https://github.com/dataesr/dsfr-plus/tree/main/src/components/Grid

Reply:
Please, you should not set the comment as resolved as long as it is not.